includes/class-wc-stripe-helper.php

```diff
@@ -597,4 +597,25 @@ public static function convert_wc_locale_to_stripe_locale( $wc_locale ) {
 		// Default to 'auto' so Stripe.js uses the browser locale.
 		return 'auto';
 	}
+
+	/**
+	 * Checks if this page contains a cart or checkout block.
+	 *
+	 * @since 5.3.0
+	 * @return boolean
+	 */
+	public static function has_cart_or_checkout_block_on_current_page() {
```

Comment:
Should we remove this function since it's no longer being used?

Reply:
Yes, removed in ed27e97a734d59171aac0c469dfc861533fba865

Nice catch! 👏